cdc/capture/capture.go

```diff
@@ -542,27 +569,73 @@ func (c *Capture) AsyncClose() {
 }
 
 // Drain removes tables in the current TiCDC instance.
-func (c *Capture) Drain() <-chan struct{} {
-	c.liveness.Store(model.LivenessCaptureStopping)
-	log.Info("draining the capture, removing all tables in the capture")
+func (c *captureImpl) Drain(ctx context.Context) <-chan struct{} {
+	log.Info("draining capture, removing all tables in the capture")
 
-	// Disable drain for now.
-	//
-	// TODO close done when
-	// 1. there is no table in the capture, and
-	// 2. the capture is not owner.
+	const drainInterval = 100 * time.Millisecond
```

Comment:
the `drainInterval` looks too small, how about set as 1 seconds?

Reply:
I think 100ms is okay, as it does not have heavy CPU workload nor IO workload.